### DISCUSSION TITLE
Issue with reinstall

### DISCUSSION THREAD

**Discussion by techlogik** [valinet/ExplorerPatcher#178] [General]
After installing, then following the procedure to rename dxgi.dll, reboot, delete the file, then delete the %appdata% folder, you drop the dxgi.dll file again into the c:\windows folder, kill explorer and restart it, it doesn't redownload and reinstall the extension or download and recreate the appdata folder.

**Comment by valinet** [comment]
If the symbol data is still in the registry, yeah, it doesn't do that. The registry tells it its state. Delete the entries at `HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher` and then it should behave like on a clean install.

**Reply by techlogik** [reply]
Thank you. Worked.
